src/classes/CRLP_Rollup_SEL.cls

```diff
@@ -165,9 +183,257 @@ public class CRLP_Rollup_SEL {
      * @return requested rollup__mdt
      */
     public static Rollup__mdt getRollupById(Id id) {
-
         return getRollupsById().get(id);
+    }
 
+    /**
+     * @description Returns metadata's SObject for relationshipName's QualifiedApiName with null checking.
+     * @param  metadata Metadata SObject with Metadata Relationship
+     * @param  metadataRelationship Metadata Relationship of metadata's SObject
+     * @return Tries to return metadata's SObject for metadataRelationship's QualifiedApiName.  Else, returns null
+     */
+    public static String getQualifiedApiName(SObject metadata, SObjectField metadataRelationship) {
```

Comment:
New method to collect Metadata's relationship's `QualifiedApiName` that does null checking.

Reply:
This method is removed after the refactor to only check if a Rollup Field is supported by Rollup__mdt.Operation__c (and not also by Rollup__mdt.Time_Bound_Operation_Type__c).